website/docs/index.html.markdown

```diff
@@ -102,4 +102,14 @@ The following arguments are supported:
 
 ## Testing
 
-Credentials must be provided via the `ARM_SUBSCRIPTION_ID`, `ARM_CLIENT_ID`, `ARM_CLIENT_SECRET`, `ARM_TENANT_ID` and `ARM_TEST_LOCATION` environment variables in order to run acceptance tests.
+The following Environment Variables must be set to run the acceptance tests:
+
+~> **NOTE:** The Acceptance Tests require the use of a Service Principal - authenticating via either the Azure CLI or MSI is not supported.
+
+* `ARM_SUBSCRIPTION_ID` - The ID of the Azure Subscription in which to run the Acceptance Tests.
+* `ARM_CLIENT_ID` - The Client Secret of the Service Principal.
```

Comment:
I believe this should be 'The Client ID of the Service Principal'

Reply:
Yes 🤦‍♂️